app/services/addUser.js

```diff
@@ -0,0 +1,9 @@
+const db = require('../db');
+const sql = require('../db/sql.js');
+
+module.exports = async (username, password, salt) => {
+  return await db.query(sql`
+    insert into users (username, password, salt)
+    values (${username}, ${password}, ${salt});
+  `);
```

Comment:
When creating an object is customary to return said object with the ID. But let me review the rest of the project before doing that